app/models/item.rb

```diff
@@ -3,7 +3,13 @@ class Item < ActiveRecord::Base
 
   validates_presence_of :name, :price, :quantity, :shelf_life_days
 
-  def expires
-    Date.today + shelf_life_days.days
+  scope :valid_items, -> {where 'shelf_life_days > (Date.today - created_at).in_days'}
```

Comment:
Line is too long. [86/80]<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.